app/pages/editorg/controller.js

```diff
@@ -12,3 +12,22 @@ export const getEditOrgAccountContext = async ({ organisationId, accessToken })
 
   throw new Error(`No organisation data returned for id: ${organisationId}`);
 };
+
+export const putUpdateOrganisation = async ({ organisationId, body, accessToken }) => {
+  const data = {
+    ...body,
+    catalogueAgreementSigned: !!body.catalogueAgreementSigned,
+  };
+
+  await putData({
+    endpointLocator: 'putUpdateOrganisation',
+    options: { organisationId },
+    body: data,
+    accessToken,
+  });
+
+  // eslint-disable-next-line no-underscore-dangle
+  delete data._csrf;
```

Comment:
Are we just deleting this for the purpose of logging it below?

Reply:
yes